exosphere-shared/example-apps/docker-service/application.yml

```diff
@@ -0,0 +1,11 @@
+name: docker-app
+description: An application that relies on dockerhub images
+version: 0.0.1
+
+services:
+  public:
+    local-service:
+      docker_image: exospheredev/local-service
+      location: ./local-service
```

Comment:
Its not being used here, but it would still be beneficial to have the field here for when the time comes to deploy a service to dockerhub, we wouldn't need to generate the name again, it could just be grabbed from here. 

Overall I thought it best to allow for any configuration of having the fields, you could have both and it use the location, if you just have one it uses that method, and if you have neither, an error is signaled.

Does that seem right? or would it be better to have a user remove the `docker_image` if they want to build it themselves?